controllers/dopplersecret_controller_secrets.go

```diff
@@ -214,10 +225,16 @@ func (r *DopplerSecretReconciler) UpdateSecret(ctx context.Context, dopplerSecre
 		return fmt.Errorf("Failed to load Doppler Token: %w", err)
 	}
 
-	existingKubeSecret, err := r.GetReferencedSecret(ctx, dopplerSecret.Spec.ManagedSecretRef)
+	existingKubeSecret, err := r.GetReferencedSecret(ctx, types.NamespacedName{
```

Comment:
nit: it's a little clunky defining the struct in the function call. I'd just pull it out as a variable on the line above and pass it in.